ammo-be/src/utils/regexs/regexs.ts

```diff
@@ -0,0 +1,5 @@
+const urlRegex = new RegExp(
+    /((([A-Za-z]{3,9}:(?:\/\/)?)(?:[-;:&=+$,\w]+@)?[A-Za-z0-9.-]+|(?:www\.|[-;:&=+$,\w]+@)[A-Za-z0-9.-]+)((?:\/[+~%/.\w\-_]*)?\??(?:[-+=&;%@.\w_]*)#?(?:[.!/\\\w]*))?)/
+);
+
+export default urlRegex;
```

Comment:
In order not to have too many files, I'd suggest putting this in a ValidationUtil => isValidUrl. Careful with the name of the file : /util/ValidationUtil/**index.ts**